code/HUXt.py

```diff
@@ -1202,36 +1340,35 @@ def huxt_constants():
     Return some constants used in all HUXt model classes
     """
     twopi = 2.0*np.pi
-    daysec = 24*60*60
+    daysec = 24*60*60 *u.s
     kms = u.km / u.s
     alpha = 0.15*u.dimensionless_unscaled  # Scale parameter for residual SW acceleration
     r_accel = 50*u.solRad  # Spatial scale parameter for residual SW acceleration
-    synodic_period = 27.2753*daysec*u.s  # Solar Synodic rotation period from Earth.
+    synodic_period = 27.2753*daysec  # Solar Synodic rotation period from Earth.
     v_max = 2000*kms
+    buffertime=5*u.day
     constants = {'twopi': twopi, 'daysec': daysec, 'kms': kms, 'alpha': alpha,
-                 'r_accel': r_accel, 'synodic_period': synodic_period, 'v_max': v_max}
+                 'r_accel': r_accel, 'synodic_period': synodic_period, 'v_max': v_max,
+                 'buffertime': buffertime}
     return constants
 
 
-def radial_grid():
+def radial_grid(r_min = 30.0 *u.solRad, r_max = 240.0*u.solRad, Nr = 140):
```

Comment:
I think we need to add some checks, like r_min < r_max, and maybe make sure Nr (and resulting dr) are sensible? If dr is too small it could make the model timestep tiny and grind things to a halt, and similarly if Nr (or the number of timesteps) is too large we could get into memory problems?